src/Api/Request.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Jahuty\Api;
+
+class Request
```

Comment:
This class should probably implement `Psr\Http\Message\RequestInterface`

Reply:
💥 I removed this class, used PSR-7 type hints, and GuzzleHttps implementations instead. Thanks!